ostelco-ios-client/ViewControllers/EKYC/NRCIVerifyViewController.swift

```diff
@@ -81,7 +81,7 @@ extension NRCIVerifyViewController: NetverifyViewControllerDelegate {
     
     func createNetverifyController() {
         // Prevent SDK to be initialized on Jailbroken devices
-        if JMDeviceInfo.isJailbrokenDevice() {
+        if JumioDeviceInfo.isJailbrokenDevice() {
             print("Will not allow this from a jailbroken device")
```

Comment:
should we have some kind of error screen for this @oisinzimmermann?

Reply:
That should be a different PR